MonoGame.Framework.Content.Pipeline/MonoGame.Content.Builder.targets

```diff
@@ -36,12 +36,10 @@
 
     <!-- Get all Mono Game Content References and store them in a list -->
     <!-- We do this here so we are compatible with xbuild -->
-    <ItemGroup>
-      <ContentReferences Include="@(MonoGameContentReference)">
-        <ContentOutputDir>$(ProjectDir)%(RelativeDir)bin\$(MonoGamePlatform)\%(Filename)</ContentOutputDir>
-        <ContentIntermediateOutputDir>$(ProjectDir)%(RelativeDir)obj\$(MonoGamePlatform)\%(Filename)</ContentIntermediateOutputDir>
-      </ContentReferences>
-    </ItemGroup>
+    <CreateItem Include="@(MonoGameContentReference)"
+            AdditionalMetadata="ContentOutputDir=$(ProjectDir)%(RelativeDir)bin\$(MonoGamePlatform)\%(Filename);ContentIntermediateOutputDir=$(ProjectDir)%(RelativeDir)obj\$(MonoGamePlatform)\%(Filename)">
+        <Output TaskParameter="Include" ItemName="ContentReferences" />
+    </CreateItem>
```

Comment:
Hmm. While this fixes the error, it does not produce the correct results. From the test app we get

`
 find * | grep xnb

TestMgcb/bin/DesktopGL/Root/RootFont.xnb
TestMgcb/bin/DesktopGL/Root/RootSub/RootSubFont.xnb
TestMgcb/Content/bin/DesktopGL/Content/ContentFont.xnb
TestMgcb/Content/bin/DesktopGL/Content/ContentSub/ContentSubFont.xnb
TestShared/Content/bin/DesktopGL/SharedContent/SharedFont.xnb
bin/DesktopGL/LinkedContent/LinkedContentFont.xnb
bin/DesktopGL/LinkedContent/LinkedContentSub/LinkedContentSubFont.xnb
bin/DesktopGL/LinkedRoot/LinkedRootFont.xnb
bin/DesktopGL/LinkedRoot/LinkedRootSub/LinkedRootSubFont.xnb
`

all of those files should be in `TestMgcb/bin/DesktopGL`